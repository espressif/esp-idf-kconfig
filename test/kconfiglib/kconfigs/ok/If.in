# test the if block keyword
# This specific case is especially tricky; due to pyparsing being primarily
# non-whitespace parser, a regression when "default y" and "if A" are parsed together occurs.
mainmenu "Test if block keyword"
    menu "menumenu"

        config A
            bool "a"
            default y

        if A

            config IFA
                bool "A is true"
                default y

        endif

        if !A || B

            config IFB
                bool "B is true"
                default y

        endif

    endmenu
