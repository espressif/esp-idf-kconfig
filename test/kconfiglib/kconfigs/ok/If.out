#
# Automatically generated file. DO NOT EDIT.
# Espressif IoT Development Framework (ESP-IDF)  Project Configuration
#

#
# menumenu
#
CONFIG_A=y
CONFIG_IFA=y
# end of menumenu
